.github/workflows/sonar-cloud.yml

```diff
@@ -0,0 +1,24 @@
+name: Security Build
```

Comment:
Is this workflow redundant to this step in the unit test?
https://github.com/0xPolygon/cdk-data-availability/blob/159cb61f6cd16ccce27c21b589557ce89fec61b1/.github/workflows/test-unit.yml#L59-L63

It turns out that we are running Sonar Cloud scan twice.

Reply:
good catch, no need to run twice, thanks